_docs/installing/nssm.md

```diff
@@ -0,0 +1,17 @@
+# NSSM
```

Comment:
Because NSSM and PM2 are more along the lines of launching and keeping the bot running, I feel like these could go more in pages under Using the Bot

Reply:
Moved.